scripts/build/build.mk

```diff
@@ -12,6 +12,7 @@ MOCKS_DIR = $(CURDIR)/tests/mocks
 HTTPS_GIT := https://github.com/cosmos/cosmos-sdk.git
 DOCKER := $(shell which docker)
 PROJECT_NAME = $(shell git remote get-url origin | xargs basename -s .git)
+COSMOS_BUILD_OPTIONS := v2
```

Comment:
I think it probably makes more sense to make this just the default behavior without the build flag and have v1 be optional via a flag

Reply:
It's not possible, systemtests expects a v2 tag. And we don't want to have two version of systemtests (that is just v1.0.0)